### PR TITLE
refactor: opacity enum

### PR DESCRIPTION
**Pull request by setchy**

### CODE REVIEW

File: src/components/RepositoryNotifications.tsx [hunk 1]
```diff
@@ -55,15 +57,20 @@ export const RepositoryNotifications: FC<IRepositoryNotifications> = ({
...
+          )}
+        >
           <AvatarIcon
             title={repoName}
             url={avatarUrl}
             size="medium"
             defaultIcon={MarkGithubIcon}
           />
           <span
-            className="cursor-pointer truncate opacity-90"
```
Comment (afonsojramos) [inline src/components/RepositoryNotifications.tsx:66]: Intentional?

File: src/types.ts [hunk 2]
```diff
@@ -137,6 +137,13 @@ export enum IconColor {
   WHITE = 'text-white',
 }
 
+export enum Opacity {
+  READ = 'opacity-50',
```
Comment (afonsojramos) [inline src/types.ts:141]: What's the reasoning behind the name?

File: src/components/RepositoryNotifications.tsx [hunk 3]
```diff
@@ -55,15 +57,20 @@ export const RepositoryNotifications: FC<IRepositoryNotifications> = ({
         className="group flex items-center justify-between bg-gray-100 px-3 py-2 dark:bg-gray-darker dark:text-white"
         onClick={toggleRepositoryNotifications}
       >
-        <div className="mt-0 flex flex-1 space-x-3 overflow-hidden overflow-ellipsis whitespace-nowrap text-sm font-medium">
+        <div
+          className={cn(
+            'flex flex-1 space-x-3 overflow-hidden overflow-ellipsis whitespace-nowrap text-sm font-medium',
+            Opacity.MEDIUM,
```
Comment (afonsojramos) [inline src/components/RepositoryNotifications.tsx:63]: Does the change in opacity look okay in the icon? But I like the consistency with the other change in NotifRow

**Review by afonsojramos — APPROVED** [review]

File: src/types.ts (hunk 2 above)
Comment (setchy) [reply src/types.ts:141]: That's what we currently set a Notification Row as when it is `READ` during the delay notification state change.

It was what I had used, too, in the draft PR for seeing read notifications.

Do you have a better name suggestion?

File: src/components/RepositoryNotifications.tsx (hunk 3 above)
Comment (setchy) [reply src/components/RepositoryNotifications.tsx:63]: I think it does.  Applying it at the row level seemed to make the opacity hierarchy clearer

File: src/components/RepositoryNotifications.tsx (hunk 1 above)
Comment (setchy) [reply src/components/RepositoryNotifications.tsx:66]: yes, opacity moved up to the div level

**Pull request merged**